pandas/core/frame.py

```diff
@@ -7900,7 +7901,8 @@ def idxmin(self, axis=0, skipna=True) -> Series:
 
         See Also
         --------
-        Series.idxmin
+        Series.idxmin : Return index *label* of the first occurrence
+            of minimum of values.
```

Comment:
A bit difficult to read IMHO.  Do you think `Return index of the minimum element` is good enough? May be not so precise, but to let people know if they are interested or not in this page should probably be enough.